include/rppdefs.h

```diff
@@ -80,10 +80,11 @@ typedef enum
     RPP_ERROR_INVALID_DST_CHANNELS      = -8,
     RPP_ERROR_INVALID_SRC_LAYOUT        = -9,
     RPP_ERROR_INVALID_DST_LAYOUT        = -10,
-    RPP_ERROR_INVALID_SRC_DATATYPE     = -11,
-    RPP_ERROR_INVALID_DST_DATATYPE     = -12,
-    RPP_ERROR_INVALID_SRC_OR_DST_DATATYPE      = -13,
-    RPP_ERROR_INSUFFICIENT_DST_BUFFER_LENGTH    = -14
+    RPP_ERROR_INVALID_SRC_DATATYPE      = -11,
+    RPP_ERROR_INVALID_DST_DATATYPE      = -12,
+    RPP_ERROR_INVALID_SRC_OR_DST_DATATYPE     = -13,
```

Comment:
Please only have one return code for invalid_datatype

Reply:
Done